todo_app/app.py

```diff
@@ -1,55 +1,85 @@
-from flask import Flask, render_template, request, redirect, url_for
+from flask import Flask, render_template, request, redirect
+from flask_login import login_required, login_user, current_user
 from todo_app.ViewModel import ViewModel
-from todo_app.ApiHelper import ApiHelper
-import sys
-import os
+from todo_app.DatabaseHelper import DatabaseHelper
+from todo_app.User import User
+from oauthlib.oauth2 import WebApplicationClient
+import todo_app.login_manager as login_manager
+import requests
 
-view_model = None
+viewModel = None
 
 def create_app():
     app = Flask(__name__)
     app.config.from_pyfile('settings.py')
-    app.config['api'] = ApiHelper(app.config.get("BASE_URI"), app.config.get(
-        "QUERY"), app.config.get("BOARD_ID"))
+    app.config['api'] = DatabaseHelper(app.config.get("DATABASE_URL"), app.config.get(
+        "DATABASE_NAME"))
     api = app.config['api']
+    login_manager.login_manager.init_app(app)
+
+    def checkRole(f):
```

Comment:
The name doesn't reveal what this decorator does. Either the required role should be a parameter, or you should rename this to something like `writer_required`